sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateSchemaV3File.scala

```diff
@@ -50,7 +50,7 @@ class StateSchemaV3File(
     fileManager.mkdirs(metadataPath)
   }
 
-  private def deserialize(in: InputStream): List[ColumnFamilySchema] = {
+  private[sql] def deserialize(in: InputStream): List[ColumnFamilySchema] = {
```

Comment:
we can skip this ? I am removing this in my PR anyway

Reply:
Will need to access this function in the test suite. Let's keep it as it is and will rebase after your PR is merged.